app/Models/Ipv6Address.php

```diff
@@ -17,10 +17,15 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  *
+ * Original Code:
+ *
  * @link       https://www.librenms.org
  *
  * @copyright  2018 Tony Murray
  * @author     Tony Murray <murraytony@gmail.com>
+ *
+ * Modified by:
```

Comment:
You don't have to mark original code and modified by.  (we have git history for that)
Preferred way is just to add @author.  You can also add @copyright if you feel it is appropriate.

Reply:
hope now it is OK